modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/test/components/filter/TimeRangeFilter.es.js

```diff
@@ -109,7 +109,7 @@ describe('The time range filter component should', () => {
 			expect(dateEndInput.value).toEqual('12/09/2019');
 		});
 
-		test('Sho error span with invalid date input', () => {
+		xtest('Sho error span with invalid date input', () => {
 			fireEvent.change(dateStartInput, {target: {value: '13/09/2020'}});
```

Comment:
@matuzalemsteles I'm working on fixing these, but I also am wondering how this was "passing" in the first place: `dateStartInput` and `dateEndInput` aren't even defined here.

Reply:
Oh yeah, this is really weird. I didn't understand either, apparently, it's stated above in the `describe` but I don't see where this is being filled in 🤷‍♂️